Cargo.lock

```diff
@@ -236,6 +236,7 @@ dependencies = [
  "dialoguer",
  "diesel",
  "diesel_migrations",
+ "enum-iterator",
```

Comment:
> The hash map was not appreciated by my college.
> Lets see if he likes this...

:D
That was before we noticed the `async` though...
Adding yet another dependency just for this isn't ideal either (IMO we already have too many dependencies) but probably acceptable since I don't think we'll really manage to reduce the dependencies anyway.